### DISCUSSION TITLE
TypeError: openpgp.readArmoredKey is not a function (OpenPGP.js v4.10.10 - 2021-01-24)

### DISCUSSION THREAD

**Discussion by Firehammer047** [openpgpjs/openpgpjs#1232] [Q&A] [answered]
Hi all,
First, awesome work on this project. I am able to generate keys with openpgp.generateKey() from dist/openpgp.min.js

But once I store the key in a variable and want to use it to sign cleartext I get this error:

TypeError: openpgp.readArmoredKey is not a function

The code was taken directly from your documentation.
https://github.com/openpgpjs/openpgpjs/blob/master/README.md

Seems readArmoredKey is defined somewhere I think, but since I'm not a JS developer I'm not very familiar with how it all works.

I also found openpgp.key.readArmored() defined somewhere. 

Have you changed the functions and not updated the documentation?

Thanks for any insight in getting this to work.

**Comment by twiss** [comment, marked as answer]
Hey :wave: We recently merged the v5 branch, which will be released soon. For the documentation of v4.10.10, please see https://github.com/openpgpjs/openpgpjs/tree/v4.10.10#readme.
